pkg/cloud/rgraph/rnode/backendservice/node.go

```diff
@@ -104,7 +110,19 @@ func (n *backendServiceNode) Actions(got rnode.Node) ([]exec.Action, error) {
 		return rnode.RecreateActions[compute.BackendService, alpha.BackendService, beta.BackendService](&ops{}, got, n, n.resource)
 
 	case rnode.OpUpdate:
-		// TODO
+		gotRes := got.(*backendServiceNode).resource
+		var fingerprint string
+		switch gotRes.Version() {
+		case meta.VersionGA:
+			obj, err := gotRes.ToGA()
+			if err != nil {
+				return nil, fmt.Errorf("Update function failed, cannot fetch fingerprint form GA object, err: %v", err)
+			}
+			fingerprint = obj.Fingerprint
```

Comment:
should we have `default:` case here? It seems we will end up with an empty fingerprint if `Version()` isn't `VersionGA`

Reply:
As mentioned in other comment. Conversion form Alpha or Beta to GA should always work and on Accessing Mutable resource we invoke the postAccess function which copies all fields to other versions. It should be tested though